lib/order.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'menu'
+
+module Order
```

Comment:
Again, this should be a class, not a module.  One way to determine this is to ask, 'Does an order get created?'.  Clearly it does, so it must be a class.